### PR TITLE
Fix README.md param issue

### PR DESCRIPTION
**Pull request by iskdrews**
Fixing README docs missing param in the Running a Local Graph Node section.

### CODE REVIEW

**Review by lutter — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Once you have all the dependencies set up, you can run the following:
 ```
 cargo run -p graph-node --release -- \
   --postgres-url postgresql://USERNAME[:PASSWORD]@localhost:5432/graph-node \
-  --ethereum-rpc [URL] \
+  --ethereum-rpc [NETWORK_NAME]:[URL] \
```
Comment (otaviopace) [inline README.md:46]: I think it should be `NETWORK_NAME:[CAPABILITIES]:URL`, like it's described [here](https://github.com/graphprotocol/graph-node/blob/ff20590829230c9d3ed0751247ed39a7d567ac17/README.md?plain=1#L113).

Comment (iskdrews) [reply README.md:46]: I was following the these steps from the Indexer docs and it was presented like this: 
https://thegraph.com/docs/indexing
Screenshot from 2021-11-12 13-06-22

But I think you're right, just I want an approve on this and will add it.

Comment (otaviopace) [reply README.md:46]: Thank you so much for linking the same issue in the docs, I can fix it there ❤️

Comment (iskdrews) [reply README.md:46]: You're welcome :) 
I will edit it now to be `NETWORK_NAME:[CAPABILITIES]:URL`

**Review by otaviopace — APPROVED** [review]

**Pull request merged**